Assets/__Scripts/MapEditor/Loading/CustomPlatformsLoader.cs

```diff
@@ -0,0 +1,747 @@
+                        1, 2, 5, //face right
+	                    1, 5, 6,
+                        0, 7, 4, //face left
+	                    0, 4, 3,
+                        5, 4, 7, //face back
+	                    5, 7, 6,
+                        0, 6, 7, //face bottom
+	                    0, 1, 6
+                    };
+
+                    mesh.vertices = vertices;
+                    mesh.triangles = triangles;
+
+                    Color[] colors = new Color[vertices.Length];
+                    for (int i = 0; i < vertices.Length; i++)
+                    {
+                        colors[i] = tubeLight.color;
+                    }                    
+                    mesh.colors = colors;
+                    
+                    Vector3 offset = tubeLight.transform.position - tubeLight.transform.TransformPoint(mesh.bounds.center);
+                    tubeLight.transform.position = tubeLight.transform.position + offset;
+
+                    if (PrefabStageUtility.GetPrefabStage(mFilter.gameObject) == null)
+                    {
+                        tubeLight.gameObject.GetComponent<MeshFilter>().sharedMesh = mesh;
+                    }
+                    else
+                    {
+                        tubeLight.gameObject.GetComponent<MeshFilter>().mesh = mesh;
+                    }
+                }
+            }
+        }
+    }
+
+    private void SetShadersCorrectly(Renderer renderer)
+    {
+        Material[] materials = (PrefabStageUtility.GetPrefabStage(renderer.gameObject) == null) ? renderer.sharedMaterials : renderer.materials;
+
+        if (materials.Length >= 1)
+        {
+            for (var i = 0; i < materials.Length; i++)
+            {
+                Material tempMaterial = materials[i];
+                if ((tempMaterial?.shader?.name?.Contains("BeatSaber/Standard") ?? false) || (tempMaterial?.shader?.name?.Equals("Standard") ?? false))
+                {
+                    tempMaterial.shader = Shader.Find("Universal Render Pipeline/Simple Lit");
+                }
+                materials[i] = tempMaterial;
+            }
+        }
+
+        if (PrefabStageUtility.GetPrefabStage(renderer.gameObject) == null)
+        {
+            renderer.sharedMaterials = materials;
+        }
+        else
+        {
+            renderer.materials = materials;
+        }
+    }
+
+    private void SetRendererMaterials(Renderer renderer, LightsManager lightsManager = null, float width = 1f)
+    {
+        Material[] materials = (PrefabStageUtility.GetPrefabStage(renderer.gameObject) == null) ? renderer.sharedMaterials : renderer.materials;
+
+        if (materials.Length >= 1 && materials[0] != null)
+        {
+            if (materials[0] != null && (width >= 0.5f))
+                Array.Resize<Material>(ref materials, materials.Length + 1);
+
+            Material lastMaterial = lightMaterial;
+            for (var i = 0; i < materials.Length; i++)
```

Comment:
I'm pretty sure the code is somewhere around here, however make sure that `ControllableLight` is the only material on lights. Lights on the Camellia Platform will turn red when the controllable lights are off since their default material still exists

Reply:
From what I saw by various levels, I can't just replace the material.
For example in this level: https://modelsaber.com/Platforms/?id=1565632702&pc
If I remove the GlowWhite, every glowable part would be invisible(transparent) when not active, this would look very strange and is not the behaviour you have in beat saber, where they stay white while not on.

I know that there are maps where this looks strange, but from what I can tell this is as it should be, you normally wouldn't see it (Camellia for example) as it is dark in beat saber and you therefore don't see the color of it.

Reply:
https://imgur.com/a/VTDmbZX
As you can see here, the Lines and the Rings are still visible, even tough they are not enlightened. Maybe I should replace the _replace_X_glow with black (Except the _replace_white_glow), that would probably do the trick. What do you think?

Reply:
As long as we can get it close to how it looks in Beat Saber, then I'll be happy. I didn't like how the red of the original material stood out, and would definitely be something people would falsely flag as a bug

Reply:
Okay, I'll therefore do that then, I think this will do the trick